packages/pyright-internal/src/analyzer/service.ts

```diff
@@ -1069,8 +1039,6 @@ export class AnalyzerService {
     private _updateSourceFileWatchers() {
         this._removeSourceFileWatchers();
 
-        this._backgroundAnalysisProgram.invalidateCache();
```

Comment:
These invalidations don't do anything; this function is always run on a fresh import resolver.

Reply:
I'm fine with this change as long as HeeJae is. I'm pretty sure he added this originally, and I'm not sure what the reasoning was. I agree that there doesn't seem to be any reason why we'd need to explicitly invalidate the cache in this code path.